### PR TITLE
Export bounding boxes for carla image and video

### PR DESCRIPTION
**Pull request by swsuggs**
Allows exporting of bounding boxes for object detection scenarios

### CODE REVIEW

**Comment by lcadalzo** [comment]
@swsuggs if the black CI test is still failing after you run black, make sure you have [this](https://github.com/twosixlabs/armory/blob/master/.github/workflows/ci_test.yml#L164) version installed

File: armory/utils/export.py [hunk 1]
```diff
@@ -64,7 +93,15 @@ def _make_output_dir(self):
             )
         os.mkdir(self.output_dir)
 
-    def _export_images(self, x, x_adv):
+    def _export_images(
+        self, x, x_adv, y=None, y_pred_adv=None, y_pred_clean=None, classes_to_skip=None
+    ):
+
+        plot_boxes = True if y is not None else False
```
Comment (lcadalzo) [inline armory/utils/export.py:100]: The logic of `plot_boxes = True if y is not None else False` doesn't seem intuitive. This variable also feels redundant with `plot_bboxes`. Maybe exporting object detection should just have its own function?

File: armory/utils/export.py [hunk 2]
```diff
@@ -64,7 +93,15 @@ def _make_output_dir(self):
...
+        if classes_to_skip is not None:
+            if type(classes_to_skip) == int:
```
Comment (lcadalzo) [inline armory/utils/export.py:102]: use of `isinstance(x, y)` is preferred over `type(x) == y`

**Comment by lcadalzo** [comment]
@swsuggs ready for re-review. I've created a separate issue #1273 to track some further-reaching desired changes to the export feature. This PR simply adds the capability to view boxes for CARLA scenarios, and I think we can address the other aspects in a later PR

**Comment by swsuggs** [comment]
Are you good with leaving your other unresolved comments for 1273 or would you prefer me to address them now?

**Comment by lcadalzo** [comment]
I'm good with addressing the comments in 1273 in a later PR that does a broader refactor of exporting samples. Also, I think we ignore the TF2 failed test since it's failing on `develop` and is a know, separate issue

**Comment by swsuggs** [comment]
Cool.  I can't click the approve button since it's my own PR but everything looks good

**Pull request merged**